css/volunteer-opportunities.css

```diff
@@ -0,0 +1,429 @@
+}
+
+.volunteer-placement-item {
+  display: flex;
+  flex-direction: row;
+}
+
+.volunteer-opportunities-heading {
+  font-size: 55px;
+  color: #fff;
+  text-align: center;
+  margin: 50px 0;
+}
+
+.placement-image-container {
+  display: flex;
+  align-items: center;
+}
+
+@media (max-width: 768px) {
+  .volunteer-section {
+    margin-top: 3em;
+    background: linear-gradient(to bottom left, #2baaed, #0a5f93);
+    padding-bottom: 0px;
+    color: white;
+  }
+
+  .volunteer-opportunities-heading {
+    font-size: 2.5em;
+    color: #fff;
+    text-align: center;
+    margin: 50px 0;
+  }
+  .volunteer-opportunities-high-school-subheading {
+    text-align: left;
+  }
+
+  .volunteer-section .volunteer-subheading-section {
+    display: block;
+    text-align: left;
+  }
+
+  .volunteer-opportunities-hs-student-section {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 2em;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .volunteer-placement-item {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .volunteer-placement-item
+    .placement-image-container
+    .volunteer-placement-img {
+    margin-top: 1em;
+    width: 100%;
+    height: auto;
+    border-radius: 30px;
+  }
+
+  .placement-image-container {
+    display: flex;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+  }
+
+  .volunteer-placement-sentence {
+    margin-top: 2em;
+    text-align: left;
+  }
+
+  .volunteer-benefits-students-grid {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    color: black;
+    margin-top: 2em;
+  }
+
+  .not-volunteer-requirement {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .not-volunteer-requirement-section-container-fluid
+    .not-volunteer-requirement
+    .volunteer-grid-items {
+    margin-bottom: 2em;
+    margin-top: 1em;
+    text-align: left;
+    margin-right: 2em;
+  }
+
+  .not-volunteer-requirement-section-container-fluid {
+    text-align: left;
+  }
+
+  .volunteer-benefits-section {
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .volunteer-grid {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-left: 0em;
+  }
+
+  .volunteer-grid .volunteer-grid-item {
+    background-color: #fff;
+    border-radius: 40px;
+    box-shadow: 7px 7px 11px rgba(0, 0, 0, 0.16);
+    max-width: 500px;
+    min-width: 250px;
+    padding: 15px;
+    height: 95%;
+    color: black;
+    margin-bottom: 2em;
+    margin-right: 0em;
+  }
+
+  .community-service-req-grid-paragraph {
+    width: 100%;
+  }
+
+  .volunteer-opportunities-img {
+    width: 90%;
+    height: auto;
+    border-radius: 30px;
+  }
+
+  .volunteer-benefits-student-grid-item {
+    margin-left: 0;
+  }
+
+  #app-info-card-title {
+    padding: 0;
+  }
+
+  .container-fluid {
+    padding: 0;
+  }
+
+  .col-lg-6 {
+    padding: 0;
+    margin: 0;
+  }
```

Comment:
The names of col-lg-6 and container-fluid (line 413 to 420) sound like they are not supposed to be exclusive to volunteer-opportunities. They should be put back where you found them originally.